README.md

```diff
@@ -70,7 +70,18 @@ Create a new [Mapbox GL JS plugin](https://www.mapbox.com/blog/build-mapbox-gl-j
 modifies the layers of the map style to use the `text-field` that matches the browser language.
 
 As of Mapbox GL Language v1.0.0, this plugin no longer supports token values (e.g. `{name}`). v1.0+ expects the `text-field`
-property of a style to use an expression of the form `['get', 'name_en']` or `['get', 'name']`; these expressions can be nested.
+property of a style to use an [expression](https://docs.mapbox.com/mapbox-gl-js/style-spec/expressions/) of the form `['get', 'name_en']` or `['get', 'name']`; these expressions can be nested. Note that `get` expressions used as inputs to other expressions may not be handled by this plugin. For example:
+```
+["match", 
+  ["get", "name"], 
+  "Canada", 
+  "America's Hat", 
```

Comment:
hehe maybe we should use an example that is less offensive to canadians. @tristen do you have suggestions.

Reply:
I likey. @willymaps?

Reply:
```
["match", 
  ["get", "name"], 
  "California", 
  "Golden State", 
  ["coalesce", 
    ["get", "name_en"], 
    ["get", "name"]
  ]
]
```